src/client/bridge/gateway/intents.rs

```diff
@@ -0,0 +1,133 @@
+use bitflags::__impl_bitflags;
+use serde::{
+    de::{Deserialize, Deserializer},
+    ser::{Serialize, Serializer},
+};
+
+#[derive(Copy, PartialEq, Eq, Clone, PartialOrd, Ord, Hash)]
+/// [Gateway Intents] will limit the events your bot will receive via the gateway.
+/// By default, no intents are specified by Serenity.
+///
+/// [Gateway Intents]: https://discordapp.com/developers/docs/topics/gateway#gateway-intents
+pub struct GatewayIntents {
+    /// The flags composing gateway intents.
+    ///
+    /// # Note
+    /// Do not modify this yourself; use the provided methods.
```

Comment:
> use the provided methods

There aren't any, however, so this statement is misleading. You should add them in the same way we do for the [`Permissions` flags](https://github.com/serenity-rs/serenity/blob/current/src/model/permissions.rs#L272).

Reply:
My bad, I forgot that, they will be added of course!